eng/code-quality-reports/src/main/resources/revapi/revapi.json

```diff
@@ -148,6 +148,12 @@
           "new": "(class|interface|enum) com\\.azure\\.perf\\.test\\.core\\..*",
           "justification": "perf-test classes are allowed to be exposed."
         },
+        {
+          "regex": true,
+          "newArchive": "com\\.azure:azure-core:.*",
+          "newArchiveRole": "supplementary",
+          "justification": "Ignore dependency changes in azure-core."
```

Comment:
We should consider adding this exception eagerly for Jackson and Reactor too which has caused similar release blockers to client libraries.

Reply:
Jackson and Reactor is already being ignored wholesale: https://github.com/Azure/azure-sdk-for-java/blob/9c873861589bf0878db793cb0753f8eabe9997fd/eng/code-quality-reports/src/main/resources/revapi/revapi.json#L56